.github/workflows/release.yml

```diff
@@ -68,34 +67,26 @@ jobs:
         with:
           subject-path: "src/**/*.nupkg"
 
-  sbom:
-    runs-on: ubuntu-latest
-    permissions:
-      contents: write # upload sbom to a release
-    needs: release-please
-    continue-on-error: true
-    if: ${{ needs.release-please.outputs.release_created }}
-
-    steps:
-      - uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4
+      # Process OpenFeature project
+      - name: Generate and Attest SBOM for OpenFeature
+        uses: ./.github/actions/sbom-generator
```

Comment:
Are you able to call this action with a normal GitHub token?

Reply:
Yes, the GH token is passed as an input in the composite action.